src/quill.coffee

```diff
@@ -241,6 +282,13 @@ class Quill extends EventEmitter2
     params[3] ?= Quill.sources.API
     return params
 
+  _intersectFormats: (formatGroups) ->
+    return _.reduce(formatGroups.slice(1), (inAll, formats) ->
```

Comment:
Hi @willrowe, I was in need of this today, so thank you. 

Have tried it out but I have an issue when calling the get methods when the selection spans across formatted and unformatted text, which leaves `formats` `undefined`. Just need a `formats = {}` here? Similar to the `Toolbar` module implementation. 

```coffee
return _.reduce(formatGroups.slice(1), (inAll, formats = {}) ->
```